llvm/test/Transforms/InstCombine/trig-nofastmath.ll

```diff
@@ -0,0 +1,59 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -passes=instcombine -S | FileCheck %s
+
+define float @mytan(float %x) {
+; CHECK-LABEL: define float @mytan(
+; CHECK-SAME: float [[X:%.*]]) {
+; CHECK-NEXT:    [[CALL:%.*]] = call float @atanf(float [[X]])
+; CHECK-NEXT:    [[CALL1:%.*]] = call float @tanf(float [[CALL]])
```

Comment:
All of these tests don't have flags. You should have all the tests with the necessary flags, and then some negative tests with missing flags

Reply:
I think this is what the nofastmath path is for but okay.